torch/csrc/api/include/torch/nn/module.h

```diff
@@ -389,9 +389,9 @@ class TORCH_API Module : public std::enable_shared_from_this<Module> {
 
   /// Deserializes the `Module` from the given `InputArchive`.
   ///
-  /// If the `Module` contains unserializable submodules (e.g. `nn::Functional`),
-  /// we don't check the existence of those submodules in the `InputArchive` when
-  /// deserializing.
+  /// If the `Module` contains unserializable submodules (e.g.
+  /// `nn::Functional`), we don't check the existence of those submodules in the
+  /// `InputArchive` when deserializing.
```

Comment:
same here (and a few places in this file), would be great if we don't reformat the comments in this PR. If we choose to format with `clang-format` in the future, we should have a standalone PR just for that purpose.